conf.py

```diff
@@ -32,6 +32,14 @@
 else:
     acos_submit_base_url = 'https://acos.cs.aalto.fi'
 
+override = {
+    'jututfeedback': {
+        'url': ('http://jutut:8082/feedback/testcoursemultilang/{key}' if on_container
```

Comment:
I will change the hardcoded course key. `testcoursemultilang` does not match Aplus-manual.